js/app.js

```diff
@@ -193,15 +237,19 @@
    *   invoke this function expression passing 2 arguments
    *     event, 'light-green'
    */
-
+  btn13.addEventListener('click', function (event) {
+    setMyLightClass.bind(btn13, event, 'light-green')();
```

Comment:
The bind function should only take in one argument, the rest of the arguments should be in the subsequent call the the returned function.